plugins/gatsby-source-wagtail/test/dummy.json

```diff
@@ -0,0 +1,72 @@
+[
+  {
+    "id": 1,
+    "title": "Root",
+    "slug": "root",
+    "last_published_at": null,
+    "type": "wagtailcore.Page"
+  },
+  {
+    "id": 2,
+    "title": "Welcome to your new Wagtail site!",
+    "slug": "home",
+    "last_published_at": null,
+    "type": "wagtailcore.Page"
+  },
+  {
+    "id": 3,
+    "title": "Hogwarts School of Witchcraft and Wizardry",
+    "slug": "hogwarts-school-witchcraft-and-wizardry",
```

Comment:
Should we add the `idp_slug` here also with it being implemented now?  Or add it later?

Reply:
Good point, I'll add it to be consistent.